src/chromium/interoperability-with-cpp/error-handling-png.md

```diff
@@ -0,0 +1,67 @@
+# cxx error handling - PNG example
+
+A prototype of a PNG decoder illustrates what can be done when the successful
+result cannot be passed across the FFI boundary.
+
+Notes:
+
+* cxx doesn't support
```

Comment:
Here and throughout: let's call this "CXX" to be consistent with the spelling on https://cxx.rs/. That is, "cxx" is the crate name, and "CXX" is the project name. When at all possible, I prefer my sentences and headings to start with a capital letter, so that will fix this elegantly :smile: